my-app/src/components/error.tsx

```
@@ -0,0 +1,23 @@
+import { AlertCircle } from "lucide-react"
+
+interface Props {
+    title: string,
+    description: string
+}
+
+export const Error = ({
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)